skipruntime-ts/core/src/internals/skipruntime_module.ts

```diff
@@ -1087,9 +1161,90 @@ export class SkipRuntimeFactory implements Shared {
   }
 }
 
-class SkipRuntimeImpl implements SkipRuntime {
+export type Values<K extends TJSON, V extends TJSON> = {
+  values: Entry<K, V>[];
+  reactive?: ReactiveResponse;
+};
+
+export type GetResult<T> = {
+  request?: string;
+  values: T;
```

Comment:
```suggestion
  result: T;
```

The `.values.values` in the `getAll` helper in `runtime.spec.ts` seems likely a common pattern; this makes it more clear what those fields actually mean.